update.go

```diff
@@ -60,7 +60,13 @@ func updateCLI(channel string) {
 	if Autoupdate == "no" {
 		return
 	}
-	manifest := GetUpdateManifest(channel)
+	if config.LockChannel != "" {
+		if channel != "stable" && channel != config.LockChannel {
```

Comment:
@dickeyxxx I think this should be `channel != Channel` since that is what the default is when not passed